### PR TITLE
Bug-2088346: ovn-k needs 2 default gateways for dual-stack

### PR DESCRIPTION
**Pull request by mkowalski**
Fixes: Bug-2088346

Version(s): 4.8+

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2088346

Link to docs preview: https://deploy-preview-45862--osdocs.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.html#nw-ovn-kubernetes-limitations_about-ovn-kubernetes

### CODE REVIEW

**Comment by mkowalski** [comment]
/cc astoycos
/cc rbbratta

You worked on a previous PR to the same file. Can you please have a look here?

**Comment by mkowalski** [comment]
/cc @ahardin-rh
/cc @bpickard22 
/cc @anuragthehatter

Picking you as last reviewers for this file with OVN limitations. Change applies to 4.8 and later because 4.8 is when dual-stack started to be supported

File: modules/nw-ovn-kuberentes-limitations.adoc [hunk 1]
```diff
@@ -25,3 +25,15 @@ F1006 16:09:50.985939   60651 ovnkube.go:130] multiple gateway interfaces detect
 ----
 +
 The only resolution is to reconfigure the host networking so that both IP families use the same network interface for the default gateway.
+
+* For clusters configured for dual-stack networking, both IPv4 and IPv6 routing table must contain the default gateway.
```
Comment (mjpytlak) [inline modules/nw-ovn-kuberentes-limitations.adoc:29]:
```suggestion
* For clusters configured for dual-stack networking, both the IPv4 and IPv6 routing tables must contain the default gateway.
```

**Review by mjpytlak — COMMENTED** [review]
One small update from a peer review perspective. Otherwise LGTM.

**Review by sayjadha — COMMENTED** [review]
LGTM.

File: modules/nw-ovn-kuberentes-limitations.adoc [hunk 2]
```diff
@@ -25,3 +25,15 @@ F1006 16:09:50.985939   60651 ovnkube.go:130] multiple gateway interfaces detect
 ----
 +
 The only resolution is to reconfigure the host networking so that both IP families use the same network interface for the default gateway.
+
+* For clusters configured for dual-stack networking, both the IPv4 and IPv6 routing tables must contain the default gateway.
+If this requirement is not met, pods on the host in the `ovnkube-node` daemon set enter the `CrashLoopBackOff` state.
+If you display a pod with a command like `oc get pod -n openshift-ovn-kubernetes -l app=ovnkube-node -o yaml`, the `status` field contains more than one message about the default gateway, as shown in the following output:
```
Comment (mburke5678) [inline modules/nw-ovn-kuberentes-limitations.adoc:31]: I am guessing that you copied the previous bullet, where this was approved. Can I ask you to change the _like_ in both bullets to _such as_? The IBM Style Guide says not to use _like_ in this way. Thank you!

**Comment by mburke5678** [comment]
@mkowalski One tiny nit. Otherwise LGTM

**Comment by mkowalski** [comment]
@mburke5678 fixed

**Pull request merged**